scripts/eval_trajectory_to_lisdf.py

```diff
@@ -56,16 +59,76 @@ def _main() -> None:
     joint_names = robot.arm_joint_names
     assert len(joint_names) == joint_arr.shape[1]
 
-    # For now, we just convert the trajectory into one big JointSpacePath. In
-    # the future, we may want to chunk it up.
-    duration = args.time_per_conf * len(joint_arr)
-    command = JointSpacePath.from_waypoints_np_array(
-        joint_arr,
-        joint_names,
-        duration=duration,
-        label=args.input,
-    )
-    lisdf_plan = LISDFPlan(commands=[command], lisdf_problem="no_problem")
+    # Convert the joint array into a list of (arm joint state, gripper state)
+    # where the gripper state is GripperPosition.open or GripperPosition.close
+    # and the arm joint state is the original joints with grippers removed.
+    gripper_idxs = [robot.left_finger_joint_idx, robot.right_finger_joint_idx]
+    arm_joint_names = list(np.delete(joint_names, gripper_idxs))
+
+    def _joints_to_arm_gripper_vals(
+        joints: NDArray[np.float32]
+    ) -> Tuple[NDArray[np.float32], GripperPosition]:
+        # Determine if gripper is currently closed or open.
+        # Use arbitrary gripper idx.
+        gripper_val = joints[gripper_idxs[0]]
+        dist_to_open = abs(gripper_val - robot.open_fingers)
+        dist_to_closed = abs(gripper_val - robot.closed_fingers)
+        if dist_to_closed < dist_to_open:
+            gripper_state = GripperPosition.close
+        else:
+            gripper_state = GripperPosition.open
+        # Extract the arm joint state.
+        arm_state = np.delete(joints, gripper_idxs)
+        return (arm_state, gripper_state)
+
+    arm_gripper_states = list(map(_joints_to_arm_gripper_vals, joint_arr))
+
+    # List of LISDF commands, alternating JointSpacePath and ActuateGripper.
+    commands: List[Command] = []
+
+    # Accumulate the arm-only actions until a change in the gripper state
+    # is observed, at which point we finalize the JointSpacePath command,
+    # add an ActuateGripper command, and then start a new JointSpacePath.
+    accum_arm_states: List[NDArray[np.float32]] = []
+
+    # Helper function for converting an accum_arm_states to a JointSpacePath.
+    def _create_path_command(
+            arm_states: List[NDArray[np.float32]]) -> JointSpacePath:
+        duration = args.time_per_conf * len(arm_states)
+        arm_states_np = np.array(arm_states)
+        path_command = JointSpacePath.from_waypoints_np_array(
+            arm_states_np,
+            arm_joint_names,
+            duration=duration,
+            label=args.input,
```

Comment:
Can we give a unique label for each command? That way it's easier to debug when it goes to the server and it executes each command.